free-programming-books-tr.md

```diff
@@ -89,6 +91,7 @@
 ### Linux
 
 * [GNU Bash Başvuru Kılavuzu](http://belgeler.org/bashref/bashref.html)
+* [Gnu Linux Komutlari](https://www.fullportal.org/GNULINUX/Komutlar/GNULINUXKOMUTLAR.pdf)(PDF)
```

Comment:
Could you please type [GNU](https://www.gnu.org/) in all uppercase?
There should also be a space before the (PDF) tag.

```suggestion
* [GNU Linux Komutlari](https://www.fullportal.org/GNULINUX/Komutlar/GNULINUXKOMUTLAR.pdf) (PDF)
```